apps/storybook/src/TeamProfilePage.stories.tsx

```diff
@@ -14,6 +14,7 @@ export default {
 const props = (): Omit<ComponentProps<typeof TeamProfilePage>, 'children'> => ({
   id: '42',
   displayName: 'Ramirez, T',
+  inactiveSince: null,
```

Comment:
Shouldn't this be undefined

Reply:
I believe when we create a new team and don't input anything in `inactiveSince` or we don't set `inactiveSince` value for existing records, its value is `null`. And the idea is to always bring this field so we can check if the team is active or not :)

I think I misunderstood the comment when I first read 😛 I read as an affirmation not a question so I added this change here https://github.com/yldio/asap-hub/pull/2106/commits/93b9988fb9de2101f24124d1715422280fc21256 so the default value is not null

Reply:
I don't think we should be returning Null as we don't do it for any other response type. We use the typings to say what could be on a entity rather than the presence of null

Reply:
Oh ignore my comment above 😝 I'll change that shortly